src/main/java/de/themorpheus/edu/taskservice/database/model/TaskDoneModel.java

```diff
@@ -0,0 +1,31 @@
+package de.themorpheus.edu.taskservice.database.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table
+public class TaskDoneModel {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int taskDoneId;
+
```

Comment:
Also, is implemented in #227 you have to tag all UUIDs with a specific column annotation (just view the other files)